src/View/Http/InjectTemplateListener.php

```diff
@@ -75,6 +81,10 @@ public function injectTemplate(MvcEvent $e)
 
         $template = $this->mapController($controller);
 
+        if (false === $template) {
+            throw new RuntimeException('Invalid controller name');
```

Comment:
Please add the name of the controller

Reply:
I'm going to change the phpdoc because `mapController()` cannot return a `false` value as phpdoc says.